Sources/Sentry/Public/SentryOptions.h

```diff
@@ -397,6 +397,19 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableAutoBreadcrumbTracking;
 
+/**
+ * An array of hosts or regexes that determines if outgoing HTTP requests will get
+ * extra `trace_id` and `baggage` headers added.
+ *
+ * This array can contain instances of NSString which should match the URL (using `contains`),
+ * and instances of NSRegularExpression, which will be used to check the whole URL.
+ *
+ * The default value adds the header to all outgoing requests.
+ *
+ * @see https://docs.sentry.io/platforms/node/configuration/options/#trace-propagation-targets
```

Comment:
`m`: I think we should not point to the node docs, we should point to the apple docs.